plugins/vpc-bridge/network/network.go

```diff
@@ -44,11 +46,12 @@ type Network struct {
 
 // Endpoint represents a container network interface.
 type Endpoint struct {
-	ContainerID string
-	NetNSName   string
-	IfName      string
-	IfType      string
-	TapUserID   int
-	MACAddress  net.HardwareAddr
-	IPAddresses []net.IPNet
+	ContainerID  string
+	NetNSName    string
+	IfName       string
+	IfType       string
+	TapUserID    int
+	PortMappings []config.PortMapping
```

Comment:
Can we please add the new PortMappings field to the end of the struct? It logically belongs together with IPAddresses because one and only one of those two fields should be set.

Reply:
Yes sure. Doing that in the next revision.